src/index.css

```
@@ -1,13 +1,62 @@
+body {
+    background-color: #2f3136;
+    color: rgb(220, 221, 222);
+}
+.App {
```

I gotta set up a css linter to enforce my preferences

```suggestion
.app {
```

---

Hey, that's just what it's called! I prefer lower-cased class names too.

---

Alright, I had somehow assumed that the class name had to be like that because the component is called "App", but apparently that's nonsense. Fixed.